.spelling

```diff
@@ -0,0 +1,26 @@
+# markdown-spellcheck spelling configuration file
+# Format - lines beginning # are comments
+# global dictionary is at the start, file overrides afterwards
+# one word per line, to define a file override use ' - filename'
+# where filename is relative to this configuration file
+stateful
+webhooks
+webhook
+websocket
+ - blog/2024-09-13-first-documenation-iteration.md
+Fishjam
+SDK
+ - guide/integration.md
+backend
+SDKs
+url
+params
+NodeJS
+e.g.
+js-server-sdk
+ - README.md
+Docusaurus
+ - guide/react-native/connecting.mdx
+pre-configured
+ - guide/react-native/installation.mdx
+TabItem
```

Comment:
**suggestion (non-blocking):** We could move all of those to global scope. It would make writing new docs less tedious.

Reply:
i didn't notice I added it locally at first, moved everything to global scope